View/CsvView.php

```diff
@@ -293,8 +293,15 @@ protected function _generateRow($row = null) {
 		$newline = $this->viewVars['_newline'];
 
 		$row = str_replace(array("\r\n", "\n", "\r"), $newline, $row);
-		if (fputcsv($fp, $row, $delimiter, $enclosure) === false) {
-			return false;
+		if ($enclosure == '') {
```

Comment:
If you modify your branch (add the fix commit on top) this PR gets updated automatically.